src/components/Modal/EditSubkeywordModal.jsx

```diff
@@ -0,0 +1,191 @@
+                  placeholder="새롭게 추가할 키워드를 입력해주세요"
+                />
+                <div className="flex items-center gap-5 w-full">
+                  {subkeywordList.includedKeyword.map((subkeywordName) => {
+                    return (
+                      <KeywordChip
+                        key={subkeywordName}
+                        keywordName={subkeywordName}
+                        hasCloseButton={true}
+                        handleCloseIconClick={() =>
+                          removeSubkeyword("includedKeyword", subkeywordName)
+                        }
+                      />
+                    );
+                  })}
+                </div>
+                <p className="text-12 text-red-500 h-18 font-semibold">
+                  {errorMessage.includedKeyword}
+                </p>
+              </div>
+              <PlusIcon
+                className="size-40 flex-shrink-0 fill-purple-300 cursor-pointer"
+                onClick={() => handleCreateSubeywordButtonClick("includedKeyword")}
+              />
+            </div>
+            <div className="w-full flex items-start gap-20">
+              <Label
+                htmlFor="excludedKeyword"
+                styles="w-110 text-20 text-violet-900 font-semibold flex-shrink-0"
+              >
+                제외 키워드:
+              </Label>
+              <div className="flex flex-col justify-start gap-3 w-full">
+                <input
+                  type="text"
+                  id="excludedKeyword"
+                  value={inputValue.excludedKeyword}
+                  onChange={(e) => handleSubkeywordInputChange(e, "excludedKeyword")}
+                  className="w-full h-40 px-15 border-2 border-purple-300 rounded-[8px] text-purple-900 font-semibold"
+                  placeholder="제외할 키워드를 입력해주세요"
+                />
+                <div className="flex items-center gap-5 w-full">
+                  {subkeywordList.excludedKeyword.map((subkeywordName) => {
+                    return (
+                      <KeywordChip
+                        key={subkeywordName}
+                        keywordName={subkeywordName}
+                        hasCloseButton={true}
+                        handleCloseIconClick={() =>
+                          removeSubkeyword("excludedKeyword", subkeywordName)
+                        }
+                      />
+                    );
+                  })}
+                </div>
+                <p className="text-12 text-red-500 h-18 font-semibold">
+                  {errorMessage.excludedKeyword}
+                </p>
+              </div>
+              <PlusIcon
+                className="size-40 flex-shrink-0 fill-purple-300 cursor-pointer"
+                onClick={() => handleCreateSubeywordButtonClick("excludedKeyword")}
+              />
+            </div>
+            <CreateKeywordButton buttonText={"저장하기"} />
```

Comment:
```suggestion
            <CreateKeywordButton buttonText={SAVE_BUTTON}/>
```
"저장하기"와 다른 파일에 있는 "생성하기" 모두 상수화시키는 것은 어떨까요?